src/tray.ts

```diff
@@ -14,134 +14,116 @@ import { t } from '@/i18n';
 
 import type { MenuTemplate } from './menu';
 
-// Prevent tray being garbage collected
 let tray: Electron.Tray | undefined;
 
 type TrayEvent = (
   event: Electron.KeyboardEvent,
   bounds: Electron.Rectangle,
 ) => void;
 
-export const setTrayOnClick = (fn: TrayEvent) => {
-  if (!tray) {
-    return;
+interface AppWindowControls {
```

Comment:
Is there a reason why you wrote this type? `getSongControls` already returns the correct type.

Reply:
Thanks for informing. Removed.